pkg/scheduler/framework/plugins/nodelabel/node_label.go

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Deprecated: this plugin has been deprecated and is only configurable through
```

Comment:
maybe we can also add a log line warning about the use of a deprecated plugin in v1beta1.

Reply:
ack. I've kept the message simple.